### PR TITLE
Feat/immudb tools

### PR DESCRIPTION
**Pull request by SimoneLazzaris**
Added references to immudb-tools

### CODE REVIEW

File: src/master/production/index-maintenance.md [hunk 1]
```diff
@@ -109,12 +109,61 @@ If you're using another development language, please refer to the [immugw](../co
...
+- online compaction
+- percentage compaction
+- full flush
+
+In all three modes, new indexes are calculated and old one are discarded. Indexes are organized in chunk files; each time a file only contains discarded indexes, it is automatically deleted.
+
+:::: tabs
+::: tab Online compaction
+
+This kind of compaction is performed by immudb during normal write operations: once the new written data reaches the percentage threshold configured per one database, immudb rebuilds its indexes up to the same percentage, discarding old ones.
```
Comment (byo) [inline src/master/production/index-maintenance.md:147]:
```suggestion
This kind of compaction is performed by immudb during normal write operations: once the amount of new written data reaches the percentage threshold configured per one database, immudb cleans up specified percentage of the index data, discarding old unreferenced data.
```

File: src/master/production/index-maintenance.md [hunk 2]
```diff
@@ -109,12 +109,61 @@ If you're using another development language, please refer to the [immugw](../co
...
+
+For every database, users can specify a percentage of total written data to be reindexed on every write.
+
+This tool can be used to enable this mode, and to set the percentage threshold. Once this is done, there is no need to run this tool: the compaction will happen automatically.
```
Comment (byo) [inline src/master/production/index-maintenance.md:151]:
```suggestion
The compactor tool can be used to enable this mode, and to set the percentage threshold. Once this is done, there is no need to run compactor tool periodically: the compaction will happen automatically.
```

File: src/master/production/replication.md [hunk 3]
```diff
@@ -122,6 +122,29 @@ Replication is configured per database. Thus, the same immudb server may hold se
...
+<WrappedSection>
+### Replicator tool
+
+You may need to keep a copy of every database on one immudb instance on another, so that when a new database is created
+on the main instance, a replicated database is created on the replica.
+
+In that case you can use the [replicator tool](https://github.com/codenotary/immudb-tools/tree/main/replicator), part of the
+[immudb tools](https://github.com/codenotary/immudb-tools).
+
+This tool connects to two immudb instance, one main instance and a replica. Periodically, scans the list of databases
```
Comment (byo) [inline src/master/production/replication.md:134]:
```suggestion
This tool connects to two immudb instances, one main instance and a replica. Periodically, scans the list of databases
```

**Review by byo — COMMENTED** [review]
Few small comments left.

File: src/master/production/index-maintenance.md [hunk 4]
```diff
@@ -109,12 +109,61 @@ If you're using another development language, please refer to the [immugw](../co
...
+
+The maintenance can be performed in three different ways:
+- online compaction
+- percentage compaction
+- full flush
+
+In all three modes, new indexes are calculated and old one are discarded. Indexes are organized in chunk files; each time a file only contains discarded indexes, it is automatically deleted.
+
+:::: tabs
+::: tab Online compaction
```
[the remark this answers is not in the document]
Comment (SimoneLazzaris) [reply src/master/production/index-maintenance.md]: Fixed in commit 30bce89ca

**Review by byo — APPROVED** [review]

**Pull request merged**